packages/client/src/chat.ts

```diff
@@ -699,3 +710,17 @@ export function updatePeopleTyping(): void {
   chat1.html(msg);
   chat2.html(msg);
 }
+
+function getChatMessage(msg: string, type: SelfChatMessageType): string {
+  let message = msg;
+  switch (type) {
+    case SelfChatMessageType.info:
```

Comment:
please use { } brackets around the cases of all switch statements to match the existing code style